tests/cpu_block_tests/tx_top_sim/test_tx_top_sim.py

```diff
@@ -11,5 +11,5 @@ def test_sim():
     DragonTester(
         ext_srcs=deps,
         directory=BUILD_DIR,
-        # dump_waveforms=True,
+        dump_waveforms=True,
```

Comment:
If you write ``dump_waveforms=dump_waveforms``, and then change the ``def test_sim():`` line to ``def test_sim(dump_waveforms)`` (i.e., adding an argument called ``dump_waveforms``), then you can enable waveform dumping from the command line when you run ``pytest``, e.g.
```
pytest -s --dump_waveforms
```
This works because of the way ``pytest`` is configured in ``conftest.py`` (where the custom argument ``dump_waveforms`` is defined)